tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -170,6 +170,49 @@ def test_cpu_template(test_microvm_with_ssh, network_config, cpu_template):
                                     "XSAVEC": "false", "XGETBV": "false",
                                     "XSAVES": "false"}
 
+    enabled_features_cpuid = {"x87": "true", "RDTSCP": "true",
```

Comment:
There are a couple of feature that I'm not sure to what CPUID flag they belong or if there is a corresponding CPUID flag. For example: 
```
kvmclock available at MSR 0x11,
kvmclock available a MSR 0x4b564d00,
async pf enable available by MSR
```
If they belong to a KVM specific CPUID leaf, I don't think we need to test them.

Anyway could you give more details about this features please ? 

Also adding a comment to each feature related to the CPUID flag that it belongs to would make it easier to check these. But I'm not sure how easy/feasible it would be.

Reply:
I have removed hypervisor related features. Also I have tried to group them as much as possible